packages/common/src/dotlottie-player.ts

```diff
@@ -1238,217 +1237,46 @@ export class DotLottiePlayer {
     try {
       this.setCurrentState(PlayerState.Loading);
 
-      const srcParsed = DotLottiePlayer.parseSrc(this._src);
+      if (typeof this._src === 'string') {
+        if (isValidLottieString(this._src)) {
+          const validLottieJson = JSON.parse(this._src);
 
-      if (typeof srcParsed === 'string') {
-        const { activeAnimationId, animations, manifest, themes } = await this.getAnimationData(srcParsed);
+          this._dotLottieLoader.loadFromLottieJSON(validLottieJson);
+        } else {
+          const url = new URL(this._src, window.location.href);
```

Comment:
We might have SSR issue for react-player.  Can we check if `URL` and `window` exits.

Reply:
works